spec.md

```diff
@@ -1103,6 +1346,16 @@ If a `CreateVolume` operation times out, leaving the CO without an ID with which
 
 It is NOT REQUIRED for a controller plugin to implement the `LIST_VOLUMES` capability if it supports the `CREATE_DELETE_VOLUME` capability: the onus is upon the CO to take into consideration the full range of plugin capabilities before deciding how to proceed in the above scenario.
 
+##### `CreateSnapshot`, `DeleteSnapshot`, `ListSnapshots`
+
+The plugin-generated `snapshot_id` is a REQUIRED field for the `DeleteSnapshot` RPC, as opposed to the CO-generated snapshot `name` that is REQUIRED for the `CreateSnapshot` RPC: these fields MAY NOT contain the same value.
+If a `CreateSnapshot` operation times out, leaving the CO without an ID with which to reference a snapshot, and the CO also decides that it no longer needs/wants the snapshot in question then the CO MAY choose one of the following paths:
+
+1. Execute the `ListSnapshots` RPC to possibly obtain a snapshot ID that may be used to execute a `DeleteSnapshot` RPC; upon success execute `DeleteSnapshot`.
```

Comment:
Limiting to these two options is problematic.
I do not recall any discussion on the last call that an idempotent `CreateSnapshot` is problematic. Unless I'm mistaken, it should be an expectation that a timed out `CreateSnapshot` will result in a subsequent `CreateSnapshot` expecting the same resource.

Reply:
Ah ok, I re-read spec here and it makes sense. Thanks!